core/src/main/java/hudson/Main.java

```diff
@@ -83,7 +86,8 @@ private static String getHudsonHome() {
     }
 
     /**
-     * Run command and place the result to a remote Hudson installation
+     * Run command and send result to {@code ExternalJob} in the {@code external-monitor-job} plugin.
+     * Obsoleted by {@code SetExternalBuildResultCommand} but kept here for compatibility.
```

Comment:
Well I figure if you care enough to open that plugin’s sources you will not have difficulty finding it.